gemrb/GUIScripts/LUProfsSelection.py

```diff
@@ -221,8 +221,10 @@ def SetupProfsWindow (pc, type, window, callback, level1=[0,0,0], level2=[1,1,1]
 			# don't give too many points; exact formula unknown, but this works with f/m and f/m/t
 			ProfsPointsLeft += sum(level2)/IsMulti[0]/ProfsRate
 		else:
-			# sum the levels, since the rate is for the combined multiclass
-			ProfsPointsLeft += (sum(level2) - sum(level1))/ProfsRate
+			# average the levels, since the rate is for the combined multiclass
+			# FIXME: should give points when the fasted-profs-rate class (f > c/t > m) levels,
+			# instead of using the average level
+			ProfsPointsLeft += sum(level2)/IsMulti[0]/ProfsRate - sum(level1)/IsMulti[0]/ProfsRate
```

Comment:
Please provide exact test cases, so they can be added to the comments or at least the commit description. It's great for when you look at this again in 5 years.

Reply:
I did describe the logic in the commit, but for exact examples, let's take a thief/mage.  She is supposed to get profs at T4/M3 and T8/M7, since thief is the proficiency-gaining class here.  With my fix she gets profs at T5/M4 and T8/M8, as I used the average level.  As I noted, it's _slightly_ incorrect.  Without my fix, she would not get profs at all when leveling one level at a time due to rounding errors, but if leveled from T1/M1 to T8/M8 all at once, she would gain _three_ profs instead of two.

Reply:
Thanks, I now pushed a fix that takes the fastest rate into account.

Reply:
Does it work correctly for cleric/thieves?  They're tricky because clerics and thieves have the same prof rate, but different XP level growth.  It seems the original BG2 gives a prof point when the cleric subclass levels, although I'm not sure why -- switching their XP requirements does not change it, so it's not because cleric levels last, at least.

Reply:
I didn't try, but from memory I'd guess it uses the cleric one, simply because they come earlier in the table.